Resources/Locale/en-US/prototypes/access/accesses.ftl

```diff
@@ -37,3 +37,43 @@ id-card-access-level-maintenance = Maintenance
 id-card-access-level-external = External
 
 id-card-access-level-nuclear-operative = Nuclear Operative
+
```

Comment:
You can also probably remove these if you're not showing the access level anymore